lib/resolveTaskFn.js

```diff
@@ -2,14 +2,17 @@ import { redBright, dim } from 'colorette'
 import execa from 'execa'
 import debug from 'debug'
 import { parseArgsStringToArgv } from 'string-argv'
+import pidTree from 'pidtree'
 
 import { error, info } from './figures.js'
 import { getInitialState } from './state.js'
 import { TaskError } from './symbols.js'
 
+export const FAIL_CHECK_INTERVAL = 200
```

Comment:
Does this need to be exported?

Reply:
Nope, that's just a leftover from a unit test that I realized isn't really meaningful.